src/Scrollable/_variables.scss

```diff
@@ -0,0 +1,33 @@
+
+// Scrollable
+
+// Padding applied to the scrollable body
+$scrollable-top-margin:                  0.46rem !default;
+$scrollable-bottom-margin:               5.4rem !default;
+$scrollable-inner-padding:               1.58rem !default;
+$scrollable-shadow-padding:              0.96rem !default;
+
+// shadow positioning
+$scrollable-bottom-shadow-position:      9px !default;
+
+// Margin between elements in footer, must be lower than or equal to 2 * $scrollable-inner-padding
+$scrollable-footer-margin-between:       .5rem !default;
+
+$scrollable-content-color:               null !default;
+$scrollable-content-bg:                  $white !default;
+$scrollable-content-border-color:        rgba($black, .2) !default;
+$scrollable-content-border-width:        0 !default;
+$scrollable-content-border-radius:       $border-radius-lg !default;
+$scrollable-content-inner-border-radius: subtract($scrollable-content-border-radius, $scrollable-content-border-width) !default;
+$scrollable-content-box-shadow-xs:       0 .25rem .5rem rgba($black, .5) !default;
+$scrollable-content-box-shadow-sm-up:    0px 10px 20px rgba($black, 0.15), 0px 8px 20px rgba($black, 0.15) !default;
+
+$scrollable-xl:                          1140px !default;
+$scrollable-lg:                          800px !default;
+$scrollable-md:                          500px !default;
+$scrollable-sm:                          400px !default;
+
+$scrollable-fade-transform:              translate(0, -50px) !default;
+$scrollable-show-transform:              none !default;
+$scrollable-transition:                  transform .3s ease-out !default;
+$scrollable-scale-transform:             scale(1.02) !default;
```

Comment:
It looks like most of these are unused and should be removed. I don't think the transitions of the shadow fade ins and outs need to be configurable.

Reply:
yeah ok thanks will review and clean these up